src/server/game/Spells/SpellMgr.cpp

```diff
@@ -5092,6 +5092,15 @@ void SpellMgr::LoadSpellInfoCorrections()
         });
     });
 
+    // Throw Axe
+    ApplySpellFix({ 196876 }, [](SpellInfo* spellInfo)
+    {
+        ApplySpellEffectFix(spellInfo, EFFECT_2, [](SpellEffectInfo* spellEffectInfo)
+        {
+            spellEffectInfo->Effect = SPELL_EFFECT_NONE;
```

Comment:
i dont like this, are we sure 196959 is not casted anywhere?

Reply:
yes, its only used with this npc but the triggered spell is not on sniffs, so i prevent the spam on the console for the non existing areatrigger